rust/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "keyvi"
+version = "0.0.1"
```

Comment:
I get the semantics behind that version number. It would still make sense to have a unified version number.